src/main/java/com/thebois/views/ColonyView.java

```diff
@@ -1,42 +1,110 @@
+        ROLE_COLORS.put(RoleType.FISHER, Color.BLUE);
+        ROLE_COLORS.put(RoleType.GUARD, Color.BLACK);
+        ROLE_COLORS.put(RoleType.LUMBERJACK, Color.BROWN);
+        ROLE_COLORS.put(RoleType.MINER, Color.GRAY);
+    }
+
+    private final Color beingColor = Color.WHITE;
+    private final float tileSize;
+    private final int radius;
     private IBeingGroup colony;
+    private Texture beingTexture;
+
+    /**
+     * Instantiates a new ColonyView.
+     *
+     * @param tileSize The size of a single tile, in pixels.
+     */
+    public ColonyView(final float tileSize) {
+        this.tileSize = tileSize;
+        this.radius = Math.round(tileSize / 2f);
+
+        createBeingTexture();
+    }
+
+    private void createBeingTexture() {
+        final int superSampledRadius = radius * TEXTURE_SUPER_SAMPLING_FACTOR;
+        final Pixmap pixmap = new Pixmap(superSampledRadius * 2,
+                                         superSampledRadius * 2,
+                                         Pixmap.Format.RGBA8888);
+        pixmap.setColor(beingColor);
+        pixmap.fillCircle(superSampledRadius, superSampledRadius, superSampledRadius);
+        beingTexture = new Texture(pixmap, true);
+        beingTexture.setFilter(Texture.TextureFilter.MipMapLinearLinear,
+                               Texture.TextureFilter.MipMapLinearLinear);
+        pixmap.dispose();
+    }
 
     /**
      * Updates the views with current beings.
      *
      * @param beingGroup list of beings to draw.
      */
-    public void update(IBeingGroup beingGroup) {
+    public void update(final IBeingGroup beingGroup) {
         this.colony = beingGroup;
     }
 
     @Override
-    public void draw(final ShapeRenderer batch) {
+    public void draw(final Batch batch, final float offsetX, final float offsetY) {
         float posX;
         float posY;
-        final float radius = 5;
 
         if (colony != null) {
-            for (IBeing pawn : colony.getBeings()) {
-                posX = pawn.getPosition().getPosX();
-                posY = pawn.getPosition().getPosY();
+            for (final IBeing pawn : colony.getBeings()) {
+                posX = pawn.getPosition().getPosX() * tileSize;
+                posY = pawn.getPosition().getPosY() * tileSize;
 
-                batch.setColor(Color.WHITE);
-                batch.circle(posX, posY, radius);
+                if (pawn.getRole() != null) {
+                    batch.setColor(ROLE_COLORS.get(pawn.getRole().getType()));
+                }
+                else {
+                    batch.setColor(Color.WHITE);
+                }
+
+                batch.draw(beingTexture,
+                           offsetX + posX,
+                           offsetY + posY,
+                           radius * 2,
+                           radius * 2);
             }
         }
     }
 
+    @Override
+    public void dispose() {
+        beingTexture.dispose();
```

Comment:
Should the view handle the disposing or gamescreen?

Reply:
Since the texture here is created in, and only ever used in, `ColonyView`, it should be `ColonyView` that is responsible for disposes of it, no? 

`GameScreen` on the other hand, is responsible for disposing of the `Stage` it contains. The `Stage`, in turn, is responsible for disposing the `Widgets` inside of it (that is, `InfoView` and `GameView`), who are then responsible for disposing of their `IView`s, who are then in turn responsible for disposing of any textures they use. 

I like the chain-pattern here, it's kind of elegant, don't you think?